### DISCUSSION TITLE
阿里云盘播放flv

### DISCUSSION THREAD

**Discussion by ghost** [AlistGo/alist#190] [Q&A] [answered]
我在我的个人网站上使用阿里云盘的外链播放视频。已经添加了
```html
 <meta name="referrer" content="no-referrer" />
```
但是MP4格式播放正常，flv格式播放仍会是，引用站点策略: no-referrer-when-downgrade，报403错误。

网站链接：[alist-flv](https://alist.liulp.club/video/?%e5%86%b0%e5%86%b0-%e6%88%91%e7%9a%84%e8%a1%8c%e6%9d%8e%e7%ae%b1.flv&flv)、[alist-mp4](https://alist.liulp.club/video/?2020%ef%bc%8c%e5%83%8f%e4%b8%80%e5%9c%ba%e7%94%b5%e5%bd%b1.mp4&mp4)

**Comment by xhofe** [comment, marked as answer]
flvjs需要单独设置referrerPolicy，参考：
https://github.com/Xhofe/alist-web/blob/v2/src/pages/list/preview/video.tsx#L51-L61

**Reply by ghost** [reply]
谢谢
